04.class/memo_app.js

```diff
@@ -78,8 +76,9 @@ export class MemosApp {
     const choices = [];
 
     for (const uuid in this.baseData) {
+      const row = this.baseData[uuid].split("\n");
       const choice = {};
-      choice.name = this.baseData[uuid][0];
+      choice.name = row[0];
```

Comment:
listでも同じようなことをやってるのでメソッド化した方がいいかも？

```suggestion
      choice.name = extractFirstLine(this.baseData[uuid]);
```

Reply:
コールバックのところもでしたが、自分では発想に至らない部分をたくさんレビューで教えていただき、この部分もなるほど！となりました👀！！
しっかり吸収していきます、ありがとうございます！！